### DISCUSSION TITLE
Cool project

### DISCUSSION THREAD

**Discussion by wachtelhund** [stormgateworld/web#100] [General]
This seems like a really cool project that could be great in time for the release of stormgate EA. Is there any continuation planned for this or is it to be archived?

**Comment by robertvanhoesel** [comment]
There is not enough API support for what we'd want for the foreseeable future. We had a great run during the first alpha, during EA we hopefully see the official API develop.

https://stormgateworld.com/discontinued/

**Reply by wachtelhund** [reply]
alright, I'll keep an eye out!
